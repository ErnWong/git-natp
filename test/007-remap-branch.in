A1----A2----A3 master
       \     \
        \     BB1----BB2----BB3 feature
         \    /      /      /
          `--B1-----B2-----B3
