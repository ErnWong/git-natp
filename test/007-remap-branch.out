A1
A2 A1
A3 A2
B1 A2
BB1 A3 B1
B2 B1
BB2 BB1 B2
B3 B2
BB3 BB2 B3
